test/fixtures/for/nested.js

```diff
@@ -1,9 +1,9 @@
 let x
-for i from 0 til 10:
-  for j from 0 til 10:
+for let i=0; i<10; i++:
```

Comment:
Regarding the conversion to `for idx i in Array(10):`, I recognize that sounds like a chore, so happy to help if you don't want to go through the drudgery